src/Workspaces/CoreTest/UtilityTest/DocumentationCommentIdTests.cs

```diff
@@ -351,5 +358,40 @@ class InnerType<B, C>
             CheckReferenceId("System.Int32[]", compilation.CreateArrayTypeSymbol(intType), compilation);
             CheckReferenceId("System.Int32*", compilation.CreatePointerTypeSymbol(intType), compilation);
         }
+
+        [Fact, WorkItem(65396, "https://github.com/dotnet/roslyn/issues/65396")]
+        public void TestInvalidTypeParameterId_CSharp()
```

Comment:
>TestInvalidTypeParameterId_CSharp [](http://example.com/codeflow?start=20&length=33)

Do these tests have to be in Workspaces layer? Can we move them into src\Compilers\Core\CodeAnalysisTest\Symbols\DocumentationCommentIdTests.cs?

Reply:
You are right, thanks. Moved: bc3da5e704b89c30eca26232352db8819a6d3647